### PR TITLE
Adds documentation example for convex_hull usage

### PR DESCRIPTION
**Pull request by Hugo-Trentesaux**
From issue #260

### CODE REVIEW

**Comment by blegat** [comment]
Thanks, can you add the file name in https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/31d59960d6edf8168bed315b4e602d42ada31f0b/docs/make.jl#L7 ?

**Comment by blegat** [comment]
Can you change the file name into "Convex hull of a set of points.jl" as this becomes the name visible in the documentation menu  ?

**Comment by Hugo-Trentesaux** [comment]
Done :)

**Comment by blegat** [comment]
Yes, Literate take care of transforming it into a notebooks

**Comment by blegat** [comment]
You need to add CDDLib here:
https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/31d59960d6edf8168bed315b4e602d42ada31f0b/docs/Project.toml#L1-L2

**Comment by Hugo-Trentesaux** [comment]
@blegat I tried to build the documentation at home but it failed with broken pipes probably due to missing system dependencies. Where can I find the list of system dependencies needed to build the documentation? Meanwhile, I added `CDDLib` inside the docs `Project.toml` file. I hope it works.

**Comment by schillic** [comment]
> Where can I find the list of system dependencies needed to build the documentation?

The Julia packages are listed here:

https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/31d59960d6edf8168bed315b4e602d42ada31f0b/docs/Project.toml#L2-L7

**Comment by Hugo-Trentesaux** [comment]
Julia package are not the problem. I was talking about the non-Julia libraries (like jupyter notebook, cdd) for which Julia only provides a binding.

**Comment by blegat** [comment]
CDDLib has `cddlib_jll` as a requirement: https://github.com/JuliaPolyhedra/CDDLib.jl/blob/master/Project.toml#L21
`cddlib_jll` contains the cddlib compiled binary, see https://pkgdocs.julialang.org/v1/artifacts/.

File: examples/Convex hull of a set of points.jl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# ## Convex hull in a plane
+
+# This examples shows how to find the convex hull in the context of a plane.
+# First we have to create an object representing the vertices. We have multiple ways of doing this
+
+using Polyhedra
+
+v = convexhull([0, 0], [0, 1], [1, 0], [0.1, 0.1]) # list of points
+v = vrep([[0, 0], [0, 1], [1, 0], [0.1, 0.1]]) # vector of points
+# number of poi nts × dimension matrix
```
Comment (blegat) [inline examples/Convex hull of a set of points.jl:12]: "poi nts" -> "points"

Comment (Hugo-Trentesaux) [reply examples/Convex hull of a set of points.jl]: https://github.com/Hugo-Trentesaux/Polyhedra.jl/commit/4bcdccdfbd9bef756800910ffdc1e4cbd5aa7b5e

**Review by blegat — APPROVED** [review]
Thanks!

**Pull request merged**